tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
     "extends": "@baltpeter/tsconfig",
     "include": ["src/**/*"],
+    "compilerOptions": {
+        "esModuleInterop": true
+    },
```

Comment:
That is redundant: https://github.com/tsconfig/bases/blob/014e8927b7155640f65a7492fb4bb09f5c49f531/bases/strictest.json#L17

Reply:
It was throwing errors for me before, but it seems like that was a caching issue.